_docs/sysadmin/configuration/course_creation.md

```diff
@@ -5,6 +5,9 @@ redirect_from:
   - /sysadmin/course_creation
 ---
 
+**Note**: If you are using Ansible for managing Submitty, 
+you can follow these [instructions](ansible_course_creation) to create a course using Ansible.
```

Comment:
you may need to specify the full path here (if it was relying on the shorter version that I've asked you to remove)

Reply:
I comitted and it seems that test still accepts the current specified path, so no need to input the entire path.